ROX-19800: Update VM reporting section for 4.3 (#67362)
openshift/openshift-docs: modules/vulnerability-management20-report-review-create.adoc

You can review the details of your vulnerability report configuration before creating it.

.Procedure

> . In the *Review and create* section, you can review the report configuration parameters, delivery destination, email template that will be used if you selected email delivery, delivery schedule, and report format. To make any changes, click *Back* to go to the previous section and edit the fields that you want to change.

Reviewer: ```suggestion
. In the *Review and create* section, you can review the report configuration parameters, delivery destination, email template that is used if you selected email delivery, delivery schedule, and report format. To make any changes, click *Back* to go to the previous section and edit the fields that you want to change.
```
